.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/aerogear/mobile-security-service
+
+    docker:
+      - image: circleci/golang:1.10
+      - image: registry.access.redhat.com/rhscl/postgresql-96-rhel7:latest
+        ports:
+          - 5432:5432
+        environment:
+          POSTGRESQL_PASSWORD: postgres
+          POSTGRESQL_USER: postgresql
+          POSTGRESQL_DATABASE: mobile-security-service
```

Comment:
Any way to not store credentials in the clear

Reply:
yes there is, they can be stored as environment variables in circleci. I don't believe it will an issue though as it is a temporary database to run integration tests. @darahayes can you confirm?

Reply:
some info on it here https://circleci.com/docs/2.0/env-vars/